hadoop-ozone/dist/src/shell/ozone/ozone

```diff
@@ -83,10 +82,6 @@ function ozonecmd_case
   RATIS_OPTS="-Dorg.apache.ratis.thirdparty.io.netty.allocator.useCacheForAllThreads=false ${RATIS_OPTS}"
 
   case ${subcmd} in
-    auditparser)
-      OZONE_CLASSNAME=org.apache.hadoop.ozone.audit.parser.AuditParser
-      OZONE_RUN_ARTIFACT_NAME="ozone-tools"
-    ;;
```

Comment:
Should we add a compatibility hack to handle the old command (without including it in the usage help)?

Something like:

```bash
OZONE_SUBCMD=$1

if [[ "$OZONE_SUBCMD" == "auditparser" ]]; then
  OZONE_SUBCMD=debug
else
  shift
fi
```